src/core/ContentTile.tsx

```diff
@@ -0,0 +1,167 @@
+import type React from "react";
+import { useCallback, useMemo } from "react";
+import Badge from "./Badge";
+import FeaturedLink from "./FeaturedLink";
+import Icon from "./Icon";
+import type { IconName } from "./Icon/types";
+import cn from "./utils/cn";
+
+type ContentTileBadge =
+  | string
+  | {
+      label: string;
+      className?: string;
+    };
+
+type ContentTileProps = {
+  title?: string;
+  className?: string;
+  description?: string;
+  cta?: {
+    text: string;
+    url: string;
+  };
+  feature?: React.ReactNode | string;
+  featureType?: "image" | "icon";
```

Comment:
if the icon is not in the ably-ui can we use the `image` if its an svg?

Reply:
https://github.com/ably/ably-ui/blob/web-4474-content-tile/src/core/ContentTile.tsx#L89-L93

If the feature type is icon then if it's a string it attempts to get it from ably-ui, but you can also give it any old react node